server/models/stage-stats.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const StageStatsSchema = new mongoose.Schema({
+  tourney: String,
+  stage: String,
+  maps: [
+    {
+      mapId: Number,
+      playerScores: [{ userId: Number, score: Number }],
```

Comment:
hmm i see this is a different type than `User.userid` which is a string, which leads to some inconsistency and requires some casting in Stat.js

we should probably either change this userid to String for consistency, or we migrate `User.userid` to Number. the latter is probably more correct, but will be more work to migrate all existing users